cwms-data-api/src/test/java/cwms/cda/data/dao/basin/BasinDaoTest.java

```diff
@@ -0,0 +1,276 @@
+package cwms.cda.data.dao.basin;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import cwms.cda.api.DataApiTestIT;
+import cwms.cda.data.dto.Location;
+import cwms.cda.data.dto.LocationIdentifier;
+import cwms.cda.data.dto.basin.Basin;
+import fixtures.CwmsDataApiSetupCallback;
+import fixtures.TestAccounts;
+import mil.army.usace.hec.test.database.CwmsDatabaseContainer;
+import org.jooq.DSLContext;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+
+class BasinDaoTest extends DataApiTestIT {
+    private static final String OFFICE_ID = TestAccounts.KeyUser.SWT_NORMAL.getOperatingOffice();
+    private static final String UNIT_SYSTEM = "SI";
+    private static final ArrayList<Location> locationsCreated = new ArrayList<>();
+    private static final Basin parentBasin = new Basin.Builder()
+            .withBasinId(new LocationIdentifier.Builder()
+                    .withName("TEST_PARENT_BASIN")
+                    .withOfficeId(OFFICE_ID)
+                    .build())
+            .withAreaUnit(UNIT_SYSTEM)
```

Comment:
I believe this is the actual area unit "m2" here not the unit system

Reply:
Fixed in latest push